applications/data_analysis/general_applied_math.ipynb

```diff
@@ -0,0 +1,907 @@
+{
```

Comment:
<p>Should <code>math</code> be referred to as a module rather than a package?</p><p> </p><p>We also probably should mention here that the numpy functions are typically much faster and optimized to work on array data (you do get to this second part a bit).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>